### DISCUSSION TITLE
What is required for a Solana token to always show up in wallets?

### DISCUSSION THREAD

**Discussion by wrux** [phantom/docs#223] [Help & Support] [answered]
### Summary

I have created an SPL 2022 token and it seems to show in some wallets, some it says 'unknown' and others doesn't show at all. It's showing correctly in Phantom for me, but shows up as `unknown` when I send the token to somebody. 

What is required for the token to always show as a verified token in wallets?

Here's my token's contract address: `GK28U5k7CNu6kyewYBeyzoogetWjiYakEhas5LNBDwmC`

I have added metadata with name, ticker, image and external links etc.

Is the token perhaps not showing because it's not listed on a DEX yet?

### Steps to Reproduce

1. Open Phantom on iOS or in browser
2. Look at tokens list (shows for me, but not for others)
3. Start a new transaction and the token shows as `unknown` for some

### Phantom Version

25.5.0

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
Hello @wrux,

For the time being the best way to ensure your token shows up in Phantom is to be included on Jupiter's Strict Token List.
